Unbound.xs

```diff
@@ -275,6 +285,12 @@ static void _close_saved_debugfd (DNS__Unbound__Context* ctx) {
         if (-1 != ctx->debugfd) close(ctx->debugfd);
 }
 
+static inline void _croak_if_pid_changed_and_threaded(pTHX_ DNS__Unbound__Context* ctx) {
+    if (ctx->threaded && (getpid() != ctx->pid)) {
+        croak("Illegal attempt to use threaded DNS::Unbound instance in subprocess");
+    }
+}
+
```

Comment:
this seems a pretty good protection
I even wonder how much damage such a behavior will do to existing code